lib/aio/jobcontext.py

```diff
@@ -55,7 +55,8 @@
 class JobContext(contextlib.AsyncExitStack):
     config: JsonObject = {}  # noqa:RUF012  # JsonObject is immutable
     logs: LogDriver
-    _forge: Forge
+    _forges: dict[str, Forge]
```

Comment:
I'd like to attach this comment to line 45, but can't. Doesn't "codeberg" need to be added to `FORGES` above?

Reply:
`FORGE_DRIVERS` sounds good!